packages/buttons/src/MenuButton.tsx

```diff
@@ -32,6 +34,7 @@ export const MenuButton = ({ color, menuItems }: MenuButtonProps) => {
         icon={<Icon as={IoEllipsisVertical} />}
         variant="unstyled"
         color={color}
+        fontSize={fontSize}
```

Comment:
can we use anything here? like: '13px' or '2em' ?
if yes I would lock it with some types so the component can be used only as expected. What do you think?
(type def on line 19)

Reply:
After more understanding of the Chakra-Theming in #44 I would now add `TypographyProps['fontSize']` from the `@chakra-ui/react` package like this:

![image](https://user-images.githubusercontent.com/3983539/121182231-2ac80080-c863-11eb-80e8-113c37886670.png)

This would allow us to use the `fontSizes` defined in the theming.

But we can also define only a view own sizes as a union type here. I'm open to all.

Reply:
At the moment I only allow `'sm' | 'md' | 'lg'`. We can always add more if needed.